modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss

```diff
@@ -32,6 +32,48 @@
   }
 }
 
+// Publications, People & Courses List landing pages
+// This keeps the gutter from growing
+// and misaligning paragraphs.
+.page-publications,
+.page-courses,
+.page-people {
+  .su-page-components {
+    div[data-react-columns="12"] {
+      &.ptype-stanford-lists,
+      &.ptype-stanford-wysiwyg,
+      &.ptype-stanford-card,
+      &.ptype-stanford-media-caption,
+      &.ptype-stanford-entity,
+      &.ptype-stanford-gallery {
+        max-width: inherit;
+
+      }
+    }
+  }
+}
+
+// News and Events list pages need a little spacing.
+.page-news,
+.page-events {
+  .su-page-components {
+    div[data-react-columns="12"] {
+      &.ptype-stanford-lists,
+      &.ptype-stanford-wysiwyg,
+      &.ptype-stanford-card,
+      &.ptype-stanford-media-caption,
+      &.ptype-stanford-entity,
+      &.ptype-stanford-gallery {
+        max-width: inherit;
+
+        @include grid-media('xl') {
+          @include padding(null null null 37px);
```

Comment:
Might it be a good idea to use a relative unit here (em, rem) instead of a fixed pixel value?

Reply:
Could be. Let me try it out.

Reply:
Fixed it.